### DISCUSSION TITLE
Multiple accounts on Windows

### DISCUSSION THREAD

**Discussion by DenisSlobodyanyuk** [vogler/free-games-claimer#273] [Ideas]
Possible to hard code multiple account for non-docker use? I would prefer docker, but since there's a captcha issue, I am using this from my Windows PC. I would like to hard code credentials for multiple accounts and possibly add the ability to have vpn change IP to prevent lockout. 
Please also add instructions on how to clear the account that was used with "node epic-games"

**Comment by vogler** [comment]
You can use different `BROWSER_DIR` for each user. No need to change IP if you stay logged in.
You could create some .bat file with
```bat
set "BROWSER_DIR=data/browser1"
node epic-games.js
set "BROWSER_DIR=data/browser2"
node epic-games.js
```
That's assuming you're already logged in, in those browser profiles which you can do in a preceding run.
Also not sure about the Windows/batch syntax - let me know if that needs to be adjusted.

To clear, you can just delete it or use a different `BROWSER_DIR`.

**Reply by naviem** [reply]
Thanks. I didn't know about this. I ended up installing the app in 3 different folders and just letting task scheduler run them 5 min after each other.
